WcaOnRails/app/assets/stylesheets/competitions.scss

```diff
@@ -150,3 +150,114 @@ $venue-map-wrapper-height: 400px;
 .big-column {
   width: 100%;
 }
+
+.competition-select.form-inline {
+  background-color: #fff;
+  width: 100%;
+  padding-bottom: 10px;
+
+  input,
+  select {
+    font-size: 12px;
+    font-weight: normal;
+    display: block;
+  }
+
+  button {
+    width: 100px;
+  }
+
+  .btn-group {
+    display: block;
+  }
+}
+
+.same-line {
+  display: inline-block;
+  margin-right: 5px;
+}
+
+.competition-list {
+  li:nth-child(odd) {
+    background-color: $table-bg-accent;
+  }
+
+  .date {
+    float: right;
+    font-weight: bold;
+  }
+}
+
+#simple-nav {
+  z-index: 10;
+  background-color: #eee;
+  line-height: 40px;
+}
+
+#lists-navigation {
+  margin: 0;
+
+  padding-left: 10px;
+
+  .last-one {
```

Comment:
You can get rid of this class and use the css [`:last-child`](https://developer.mozilla.org/en-US/docs/Web/CSS/:last-child) selector instead.